vignettes/delta-method.Rmd

```diff
@@ -1,128 +1,165 @@
-
-We'll multiply that Jacobian by the variance-covariance matrix of the regression, which captures the variance structure of the coefficients. As a quick reminder about the variance-covariance matrix, if your model contains coefficients $b_0$ to $b_n$ (each with mean $\mu_{b_i}$ and standard deviation $\sigma_{b_i}$), the $i,j$th element of the variance-covariance matrix is $cov(b_i, b_j)$.
-
-Now that we're all clear on how the pieces fit together, [Wikipedia contains a really nice derivation](https://en.wikipedia.org/wiki/Delta_method#Multivariate_delta_method)
-of the multivariate delta method.
-
-# Step-By-Step
-
-1. Calculate the Jacobian matrix of the inverse link function of $X \beta$, $J$.
-
-2. Get the variance-covariance matrix, $V$, from the regression output, or calculate it some other way.^[Usually you'll just want the variance-covariance matrix of the regression, but you'll need to modify the vanilla variance-covariance matrix if you want to cluster standard errors or something.]
-
-3. Sandwich multiply the matrices: $J^{T} V J$. You’ll end up with a $k \times 1$ matrix for the $k$ predicted levels/effects.
-
-## Calculating the Jacobian
-
-### Predictive Levels 
 
-#### Categorical or Continuous Variables
+The equations above describe how to apprxoimate predicted levels or effects, but
+why not just calculate our estimate $P(X\beta)$ directly?
 
-1. Create the covariates at which you would like predicted levels. Call these matrices $A_1, A_2, ..., A_k$. 
+Well, we can do that for the point estimate, but also want to calculate errors
+on that estimate, and the variance of $P(X\beta)$ isn't known. Fortunately, we
+_can_ calculate the variance of the approximations above. [Wikipedia contains a
+really nice 
+derivation](https://en.wikipedia.org/wiki/Delta_method#Multivariate_delta_method)
+of the multivariate delta method:
 
-1. Calculate predicted levels for each of these matrices, call each of the resulting 1 x n column vectors $q_1, q_2, ..., q_k$.
+\[\text{Var}[P(X \beta)] = \text{Var}\left[P(X_1 \beta) + \frac{d(P(X_1 \beta))}{d(X\beta)}(X\beta - X_1 \beta)\right]\]
 
-1. Calculate the derivative of the link function with respect to $X \beta$ (e.g for a simple logit with predicted level $p$, $p \cdot (1-p)$). 
+Because $X_1\beta$ is a known point, it has variance of zero, so this simplifies to
 
-1. Run each vector of predicted levels through the scalar-valued link function to end up with another 1 x n column vector. Call this $q^*_i$
+\[\text{Var}[P(X \beta)] = \text{Var}\left[\frac{d(P(X_1 \beta))}{d(X\beta)} \cdot X\beta\right]\]
 
```

Comment:
Updated this section adding a few lines showing how J' V J comes from the Taylor Series expansion above, and rearranged some of the explanation to match.